service/fcservice-postman-collection.json

```diff
@@ -66,10 +66,10 @@
 				],
 				"body": {
 					"mode": "raw",
-					"raw": "{\"jsonrpc\": \"2.0\", \"id\":\"curltest\", \"method\": \"key_derive\", \"params\": [\"equip will roof matter pink blind book anxiety banner elbow sun young\", \"m/44'/461'/0/0/0\"]}",
+					"raw": "{\n    \"jsonrpc\": \"2.0\",\n    \"id\": \"curltest\",\n    \"method\": \"key_derive\",\n    \"params\": [\n        \"robust creek govern marriage super design group accuse soccer romance shrug blame broom render apart glass expire mom enforce six range gate dose organ\",\n        \"m/44'/461'/0/0/0\",\n        \"\",\n        \"en\"\n    ]\n}",
```

Comment:
Where is this new passphrase coming from ? Maybe keep the one already used to avoid confusion.

Reply:
Fixed